packages/request-client.js/src/api/request-network.ts

```diff
@@ -26,6 +26,7 @@ import localUtils from './utils';
  */
 export default class RequestNetwork {
   public bitcoinDetectionProvider?: PaymentTypes.IBitcoinDetectionProvider;
+  public supportedIdentity: IdentityTypes.TYPE[] = Utils.identity.supportedIdentity;
```

Comment:
```suggestion
  public supportedIdentities: IdentityTypes.TYPE[] = Utils.identity.supportedIdentities;
```

Reply:
is this only an alias of `Utils.identity.supportedIdentity`? If so maybe not necessary to have it as a class field

Reply:
I kept it like in case the library and the utils have a gap.